site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -66,11 +68,51 @@ include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
 
 include::docs/v1/tech/apis/identity-providers/_links-post-response-body.adoc[]
 
+=== Request
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Link a User
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/identity-provider/link#
+--
+[WARNING.deprecated]
+====
+While this endpoint is still functional, its usage is discouraged. Please use the link:/docs/v1/tech/apis/identity-providers/links#link-a-user[Link a User endpoint] which expects an `identityProviderLink` object if your version of FusionAuth is 1.43.0 or above.
+====
```

Comment:
If we keep it documented this way, I think we should move up the deprecated warning to above the request?

Reply:
Looking at this, I think we should create a `Link a user - deprecated` and `Complete a pending link - deprecated` sections to the API. Put them at the bottom and give a complete request/response pair. @jaywood128 does that make sense?

Reply:
Sure, I can remove each and put them at the bottom. 👍